apps/generator/lib/templateConfigValidator.js

```diff
@@ -24,7 +24,7 @@ const supportedParserAPIMajorVersions = [
  * @return {Boolean}
  */
 module.exports.validateTemplateConfig = (templateConfig, templateParams, asyncapiDocument) => {
-  const { parameters, supportedProtocols, conditionalFiles, generator, apiVersion } = templateConfig;
+  const { parameters, supportedProtocols,  conditionalFiles, generator, apiVersion } = templateConfig;
```

Comment:
no extra space needed

in general this file should not be visible in PR, or?

Reply:
@derberg thanks, removed after the deprecation we need to update this